rov/sensors/imu/BNO055.py

```diff
@@ -1,4 +1,4 @@
-import BNO055
+import Adafruit_BNO055
```

Comment:
Careful here, the file is BNO055, and imports the Adafruit_BNO055 library. I wanted to name this file `Adafruit_BNO055`, but couldn't because it would clash with the library name :/